### PR TITLE
Kotlin/analytics test conversion

### PR DESCRIPTION
**Pull request by Jessica-Kuo**
AnalyticsTest.java coverage:
AT java coverage

AnalyticsTest.kt coverage:
AT kt coverage

### CODE REVIEW

File: analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt [hunk 1]
```diff
@@ -108,7 +108,7 @@ class EdgeFunctionMiddlewareTest {
         analytics.track("foo")
 
         val privateEdgeFunctions: Field = Analytics::class.java.getDeclaredField("edgeFunctionMiddleware")
-        Assertions.assertThat(privateEdgeFunctions).isNotNull
+        Assertions.assertThat(privateEdgeFunctions).isNotNull()
```
Comment (prayansh) [inline analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt:111]: good catch :)

Comment (Jessica-Kuo) [reply analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt:111]: Yay! I wanted to look at the new test and then asked Cody what to do when the error was there.

Comment (prayansh) [reply analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt:111]: surprising how the CI was passing without this 🤔

Comment (migs647) [reply analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt:111]: Yah they pass locally too without the parens, that's why I didn't catch it, but gives some weird non-compile warning.

**Review by prayansh — APPROVED** [review]
LGTM 🚢

**Pull request merged**